src/views/Point.js

```diff
@@ -16,33 +15,26 @@ import { matchBlinky } from 'components/Blinky';
 import useInvites from 'lib/useInvites';
 import { useSyncOwnedPoints } from 'lib/useSyncPoints';
 import { ROUTE_NAMES } from 'lib/routeNames';
-import { eqAddr } from 'lib/wallet';
 
 import Actions from './Point/Actions';
 import { useWallet } from 'store/wallet';
+import usePermissionsForPoint from 'lib/usePermissionsForPoint';
 
 export default function Point() {
   const history = useHistory();
   const { pointCursor } = usePointCursor();
-  const { pointCache } = usePointCache();
   const { wallet } = useWallet();
 
   const point = need.point(pointCursor);
 
-  let canAdmin = false;
-  if (point in pointCache) {
-    canAdmin = wallet.matchWith({
-      Nothing: () => false,
-      Just: wal => {
-        const pointDetails = need.fromPointCache(pointCache, point);
-        const address = need.addressFromWallet(wallet);
-        return (
-          eqAddr(address, pointDetails.owner) ||
-          eqAddr(address, pointDetails.managementProxy)
-        );
-      },
-    });
-  }
+  const { canManage } = usePermissionsForPoint(
+    // using empty string should be ok here
```

Comment:
The fact that we can't "just like use" pointCursor on a view that's centered around it is ridiculous. Maybe has gone too far. (But this is of later concern...)

Reply:
yeah totally, this is a `need.wallet` situation 👍